spec/custom/index.html

```diff
@@ -176,7 +176,7 @@ <h4 id="custom-elements-custom-tag-example">Creating a custom tag</h4>
 <p>We then need to use this class to define the element:</p>
 
 <pre class="highlight">
-document.defineElement("flag-icon", FlagIcon);</pre>
+document.registerElement("flag-icon", FlagIcon);</pre>
```

Comment:
The example should reflect the specification, not the current implementations in browsers.